_posts/2024-10-08-cockpit-326.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Cockpit 326
+author: mvo
+date: '2024-10-08'
+tags: cockpit, podman, files
+slug: cockpit-326
+category: release
+summary: 'The cockpit/ws container now supports connections to hosts without Cockpit installed and has ssh key support, cockpit-pcp packagge replaced with native PCP support in bridge, Storage gains support for the Stratis virtual filesystem, cockpit-podman supports registries without a search API, and cockpit-files gains a keyboard shortcuts dialog'
```

Comment:
That feels rather long -- then again, I don't know where these summaries actually appear on our web page. Not on the overview (https://cockpit-project.org/blog/), nor on the [feed](https://cockpit-project.org/blog/feeds/release.atom.xml). @garrett do you know? Could we just stop adding these?

Please fix the "packagge" typo.

Reply:
Fixed.